### PR TITLE
3.0: Refine multiple queues/compute resources support for pcluster configure, remove unsupported OSes

### PR DESCRIPTION
**Pull request by hanwen-pcluste**
See commit descriptions for  details

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ per cluster.
 - Remove tag: Application.
 - Prevent runtime baking, i.e. pcluster create-cluster only works for official AMIs or custom AMIs created by pcluster createami command.
 - Retain CloudWatch logs on cluster deletion by default. If you want to delete the logs during cluster deletion, set Monitoring > Logs > CloudWatch > RetainOnDeletion to False in the configuration file. 
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Remove support for CentOS8.
+- Remove support for CentOS7 on ARM.
```
Comment (demartinofra) [inline CHANGELOG.md:60]: I think we never supported Centos7 on ARM so non need to mention in changelog

File: cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py [hunk 2]
```diff
@@ -341,7 +341,7 @@ def _assert_configurations_are_equal(path_config_expected, path_config_after_inp
         )  # Test if the generated yaml can be corrected loaded by Marshmallow.
     with open(path_config_expected) as expected_conf_file:
         expected_content = yaml.safe_load(expected_conf_file)
-
+    # print(after_input_content)
```
Comment (enrico-usai) [inline cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py:344]: to be removed

File: cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py [hunk 3]
```diff
@@ -362,15 +362,18 @@ def __init__(self, aws_region_name, key, scheduler):
...
         if self.is_not_aws_batch:
             self.input_list.append(op_sys)
         self.input_list.append(head_node_instance)
         if self.is_not_aws_batch:
-            self.input_list.extend(["1", "1"])
+            self.input_list.append("1")
+        self.input_list.append("myqueue")
+        if self.is_not_aws_batch:
+            self.input_list.append("1")
         if self.is_not_aws_batch:
```
Comment (enrico-usai) [inline cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py:374]: minor: you can unify two consecutive blocks
```
if self.is_not_aws_batch:
    self.input_list.append("1")
    self.input_list.append(compute_instance)
```

File: CHANGELOG.md (hunk 1 above)
Comment (hanwen-pcluste) [reply CHANGELOG.md]: Done

File: cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py (hunk 2 above)
Comment (hanwen-pcluste) [reply cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py]: Done

File: cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py (hunk 3 above)
Comment (hanwen-pcluste) [reply cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py]: Done

**Review by enrico-usai — APPROVED** [review]

**Pull request merged**